app/src/main/java/com/example/iou/MainActivity.java

```diff
@@ -69,11 +70,11 @@ public void onPageScrollStateChanged(int state) {
         // Set the adapter onto the view pager
         viewPager.setAdapter(adapter);
 
-
-
+        // Swipes between the icons in the bottom navigation view
```

Comment:
just a small thing: you probably only need the comment above the function, since these two say the same thing!